cmd/blockchaincmd/deploy.go

```diff
@@ -126,9 +126,16 @@ redeploy the chain with fresh state. You can deploy the same Blockchain to multi
 so you can take your locally tested Blockchain and deploy it on Fuji or Mainnet.`,
 		RunE:              deployBlockchain,
 		PersistentPostRun: handlePostRun,
-		Args:              cobrautils.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			requiredArgCount := 1
```

Comment:
I believe we have a cobra related pkg/ can you move this ExactArgs replacement function there?

Reply:
good point, addressed